examples/jupyter.build.sh

```diff
@@ -33,7 +33,8 @@ acbuild set-name woofwoofinc.dog/jupyter
 acbuild run -- apt-get update -qq
 acbuild run -- apt-get upgrade -qq
 
-acbuild run -- apt-get install -qq wget bzip2
+acbuild run -- apt-get install -qq wget
```

Comment:
Just curious, what's the benefit of splitting this up? I know in Docker-land this causes additional layers to be created.